controllers/nifi_utils.go

```diff
@@ -0,0 +1,373 @@
+package controllers
```

Comment:
I wonder if it would make more sense to store this as a regular file instead of a go const and be able to copy or load it somehow so that it is easier to read and have editor support for the contents?